community/atril/APKBUILD

```diff
@@ -1,18 +1,18 @@
 # Contributor: Alan Lacerda <alacerda@alpinelinux.org>
 # Maintainer: Alan Lacerda <alacerda@alpinelinux.org>
 pkgname=atril
-pkgver=1.20.0
+pkgver=1.20.1
 pkgrel=0
-pkgdesc="A document viewer for MATE"
+pkgdesc="Document viewer for MATE"
 url="https://github.com/mate-desktop/atril"
 arch="all"
-license="GPL-2.0"
+license="GPL-2.0-or-later"
 depends="mate-icon-theme"
 # make check is broken: https://github.com/mate-desktop/atril/issues/167
 options="!check"
-makedepends="intltool glib-dev gtk+3.0-dev  mate-desktop-dev
-	dconf-dev libsm-dev libsecret-dev caja-extensions-dev caja-dev
-	poppler-dev tiff-dev itstool libxml2-utils libxml2-dev python2-dev"
+makedepends="caja-dev caja-extensions-dev dconf-dev glib-dev gtk+3.0-dev intltool
+	itstool libsecret-dev libsm-dev libxml2-dev libxml2-utils mate-desktop-dev
+	poppler-dev python2-dev tiff-dev webkit2gtk-dev"
```

Comment:
It’s not clear from this commit that you’ve added new dependency. Please split it into two commits: one “improve abuild” with pkgdesc change and sorting makdepends, and next one with upgrade and new dependency.

Reply:
Added `webkit2gtk-dev` to community/atril makedepends (build fails otherwise).